wiki/Help_centre/Tournament_bans/en.md

```diff
@@ -44,6 +44,8 @@ In rare situations, the [account support team](/wiki/People/The_Team/Account_sup
 
 You may continue to play in community tournaments that do not request official support or require screening at the discretion of the individuals organising such events.
 
+You may still staff in officially supported tournaments by streaming, commentating, or designing graphics at the host's discretion. If you wish to help with other tasks, the host of the tournament you wish to help in MUST request an exemption in their initial request for support. Depending on the reason and severity of the infringement, and with your history in mind, the [account support team](/wiki/People/The_Team/Account_support_team) may then allow this.
```

Comment:
```suggestion
You may still staff in officially supported tournaments by streaming, commentating, or designing graphics at the host's discretion. If you wish to help with other tasks, the host of the tournament you wish to help in MUST request an exemption in their initial request for support. Depending on the reason and severity of the infringement, and with your history in mind, the [account support team](/wiki/People/Account_support_team) may then allow this.
```

Reply:
Has been fixed, also took the liberty of applying the same change for the other occurences of this link on the page.